system/Session/Handlers/FileHandler.php

```diff
@@ -89,7 +89,13 @@ public function __construct($config)
 		}
 		else
 		{
-			$this->savePath = rtrim(ini_get('session.save_path'), '/\\');
+                       $sessionPath    = rtrim(ini_get('session.save_path'), '/\\');
+                       if (! $sessionPath)
+                       {
+                           $sessionPath = WRITEPATH . 'session';
+                       }
+
+                       $this->savePath = $sessionPath;
```

Comment:
this is for fallback when `App:: sessionSavePath` is null, but `ini_get('session.save_path')` returns empty string.